.github/workflows/dev.yml

```diff
@@ -56,3 +57,36 @@ jobs:
       - run: pip install -U setuptools
       - run: pip install .
       - run: adt test_config.yaml -t ${{secrets.SYNAPSE_PAT}}
+
+  ghcr-publish:
+    needs: [build, test]
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v2
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
```

Comment:
Can you elaborate on how this will end up working when building this from the dev branch? From the logs I am seeing:
```
2023-10-10T18:53:16.3467932Z   registry: ghcr.io
2023-10-10T18:53:16.3468226Z   username: BWMac
2023-10-10T18:53:16.3468881Z   password: ***
```
Would that mean only the maintainers with permissions to publish to GHCR will be able to run these steps?

Reply:
AFAIK it should use the repository itself to publish the container. The repo has Admin permissions on the container package